UI/Web/src/app/book-reader/_components/book-reader/book-reader.component.ts

```diff
@@ -1210,19 +1324,37 @@ export class BookReaderComponent implements OnInit, AfterViewInit, OnDestroy {
         this.isFullscreen = true;
         this.cdRef.markForCheck();
         // HACK: This is a bug with how browsers change the background color for fullscreen mode
-        this.renderer.setStyle(this.reader.nativeElement, 'background', this.themeService.getCssVariable('--bs-body-color')); 
+        this.renderer.setStyle(this.reader.nativeElement, 'background', this.themeService.getCssVariable('--bs-body-color'));
         if (!this.darkMode) {
           this.renderer.setStyle(this.reader.nativeElement, 'background', 'white');
         }
       });
     }
   }
 
+  updateWritingStyle(writingStyle: WritingStyle) {
+    this.writingStyle = writingStyle;
+    if (this.layoutMode !== BookPageLayoutMode.Default) {
+      const lastSelector = this.lastSeenScrollPartPath;
+      setTimeout(() => {
+        this.scrollTo(lastSelector);
+        this.updateLayoutMode(this.layoutMode);
+      });
+    } else if (this.bookContentElemRef !== undefined) {
+      const resumeElement: string | null | undefined = this.getFirstVisibleElementXPath();
+      if (resumeElement !== null && resumeElement !== undefined) {
```

Comment:
this can just be if (resumeElement)

and you can remove the typing from resumeElement as IDE can infer from getFirstVisibleElementXPath